html/pages/game_page.html

```diff
@@ -0,0 +1,18 @@
+<section class="main-section">
+    <section class="column fill-height" id="game-section">
+        <h2>Snakes And Ladders</h2>
+        <canvas class="game-space" id="canv"></canvas>
+        <section class="control-section">
+            <button class="menu-button horizontal-center">Roll Die</button>
+        </section>
+    </section>
+    <aside id="player-list rounded-container">
+        <section class="rounded-container placeholder"></section>
+    </aside>
+</section>
+<script>
+    var canvas = document.getElementById("canv");
+    var ctx = canvas.getContext("2d");
+    ctx.fillStyle = "#ffffff";
+    ctx.fillRect(0, 0, canvas.clientWidth, canvas.clientHeight);
+</script>
```

Comment:
Add a new line at end of file
Goes for the other files as well.

In VSCode:
File > Preferences > Settings
Workspace
Text Editor.Files.insertFinalNewline = true
Text Editor.Files.trimFinalNewlines = true

Please commit the .vscode/settings.json.